### DISCUSSION TITLE
WYZE CAM PAN V3- COMMAND MOVING

### DISCUSSION THREAD

**Discussion by dinosaur-lavender** [mrlt8/docker-wyze-bridge#765] [Ideas]
Support a MOVING camera WYZE PAN V3 ( COMMAND )

Thanks

**Comment by mrlt8** [comment]
Can you be a little more specific? There are a couple of existing rotary related [commands](https://github.com/mrlt8/docker-wyze-bridge/wiki/Camera-Commands).
